static/app/views/performance/newTraceDetails/virtualizedViewManager.tsx

```diff
@@ -914,6 +928,15 @@ export class VirtualizedViewManager {
 
     this.container = container;
 
+    this.container.style.setProperty(
```

Comment:
Use CSS properties and set container values at a higher level in the DOM, so that they can be shared with the ghost row